gradio/utils.py

```diff
@@ -346,3 +349,222 @@ def run_coro_in_background(func: Callable, *args, **kwargs):
     """
     event_loop = asyncio.get_event_loop()
     _ = event_loop.create_task(func(*args, **kwargs))
+
+
+client = httpx.AsyncClient()
+
+
+class Request:
```

Comment:
Should we call this class `AsyncRequest`? The name `Request` seems too generic

Reply:
since it will be used as `await Request(...)` I think it is clear that it is async. Shorter names are better imo 🐱